commcare_connect/users/models.py

```diff
@@ -69,3 +69,11 @@ class Role(models.TextChoices):
         related_name="memberships",
     )
     role = models.CharField(max_length=20, choices=Role.choices, default=Role.MEMBER)
+
+
+class ConnectIDUserLink(models.Model):
```

Comment:
An instance of this model will be created by connect at the same time it creates a user in hq for users where connect is creating their hq account.